StripePaymentSheet/StripePaymentSheet/Source/Internal/API Bindings/Link/ConsumerSession.swift

```diff
@@ -15,29 +15,39 @@ import UIKit
 final class ConsumerSession: Decodable {
     let clientSecret: String
     let emailAddress: String
+    let redactedPhoneNumber: String
     let verificationSessions: [VerificationSession]
+    let supportedPaymentDetailsTypes: Set<ConsumerPaymentDetails.DetailsType>
 
     init(
         clientSecret: String,
         emailAddress: String,
-        verificationSessions: [VerificationSession]
+        redactedPhoneNumber: String,
+        verificationSessions: [VerificationSession],
+        supportedPaymentDetailsTypes: Set<ConsumerPaymentDetails.DetailsType>
     ) {
         self.clientSecret = clientSecret
         self.emailAddress = emailAddress
+        self.redactedPhoneNumber = redactedPhoneNumber
         self.verificationSessions = verificationSessions
+        self.supportedPaymentDetailsTypes = supportedPaymentDetailsTypes
     }
 
     private enum CodingKeys: String, CodingKey {
         case clientSecret
         case emailAddress
+        case redactedPhoneNumber
         case verificationSessions
+        case supportedPaymentDetailsTypes = "supportPaymentDetailsTypes"
```

Comment:
nit: interesting - why not just have case `supportPaymentDetailsTypes` -- It's fine to keep as-is, as i'm sure it will make merging easier.

Reply:
I think they intended to make it easier to read the code, as the API name ("supportPaymentDetailsTypes") is a little clunkier.